### PR TITLE
[emr-containers] adding new features to tutorial

### PR DESCRIPTION
**Pull request by dalbhanj**
Updating the EMRContainer tutorial to include the new `configurationOverrides`
field introduced in v0.0.9.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/content/docs/tutorials/emr-on-eks-example.md [hunk 1]
```diff
@@ -85,13 +85,13 @@ Now we can go ahead and install EMR on EKS controller. First, let's export envir
 ```
 export SERVICE=emrcontainers
 export RELEASE_VERSION=`curl -sL https://api.github.com/repos/aws-controllers-k8s/$SERVICE-controller/releases/latest | grep '"tag_name":' | cut -d'"' -f4`
-export ACK_SYSTEM_NAMESPACE=ack-system
+export ACK_K8S_NAMESPACE=ack-system
```
Comment (A-Hilaly) [inline docs/content/docs/tutorials/emr-on-eks-example.md:88]: Any reason to rename this to `ACK_K8S_NAMESPACE`? i believe it is named the ACK system namespace elsewhere too..

File: docs/content/docs/tutorials/emr-on-eks-example.md [hunk 2]
```diff
@@ -252,22 +252,31 @@ aws emr-containers update-role-trust-policy \
   --role-name ${ACK_JOB_EXECUTION_ROLE}
 ```
 ## Run a Sample Spark Job
+
+Befoe running sample job, let's create CloudWatch Logs and S3 bucket to store EMR on EKS logs
```
Comment (A-Hilaly) [inline docs/content/docs/tutorials/emr-on-eks-example.md:256]:
```suggestion
Before running a sample job, let's create CloudWatch Logs and an S3 bucket to store EMR on EKS logs
```

File: docs/content/docs/tutorials/emr-on-eks-example.md [hunk 3]
```diff
@@ -314,31 +331,26 @@ Simply run these commands to cleanup your environment
...
-# configurationOverrides:
-#   applicationConfiguration:
-#   monitoringConfiguration:
-#     cloudWatchMonitoringConfiguration:
-#       logGroupName:
-#       logStreamNamePrefix:
-#     s3MonitoringConfiguration:
-#       logUri:
-```
 * You cannot delete a JobRun unless its in **error** state. There is no delete-job-run API for deleting jobs (for good reason). However, if your JobRun goes into error state, you can run `kubectl delete jobrun/<job-run-name>` to cancel the job.
```
Comment (A-Hilaly) [inline docs/content/docs/tutorials/emr-on-eks-example.md:354]: What are you planning to do with this? How does the controller reacts to JobRun deletions?

File: docs/content/docs/tutorials/emr-on-eks-example.md (hunk 1 above)
Comment (dalbhanj) [reply docs/content/docs/tutorials/emr-on-eks-example.md:88]: I tried to keep same name as mentioned in [IRSA](https://aws-controllers-k8s.github.io/community/docs/user-docs/irsa/#create-an-iam-role-for-your-ack-service-controller) setup. There seems to be regression issue bcoz both names are being used in tutorials

File: docs/content/docs/tutorials/emr-on-eks-example.md (hunk 3 above)
Comment (dalbhanj) [reply docs/content/docs/tutorials/emr-on-eks-example.md:354]: emr-containers doesn't have delete jobrun api. I'm going to create separate issue but want to hear from customers first if this is an issue for them

File: docs/content/docs/tutorials/emr-on-eks-example.md (hunk 1 above)
Comment (A-Hilaly) [reply docs/content/docs/tutorials/emr-on-eks-example.md:88]: IMO It's the IRSA setup guide that should change. `ACK_SYSTEM_NAME` is what have been used previously (with CARM and other feature)

**Pull request merged**